presto-main/src/main/java/com/facebook/presto/operator/TableFinishInfo.java

```diff
@@ -34,8 +32,8 @@
 {
     private static final int JSON_LENGTH_LIMIT = toIntExact(new DataSize(10, MEGABYTE).toBytes());
     private static final JsonCodec<Object> INFO_CODEC = jsonCodec(Object.class);
-    private static final JsonCodec<JsonNode> JSON_NODE_CODEC = jsonCodec(JsonNode.class);
 
+    private final ConnectorOutputMetadataWrapper metadataWrapper;
```

Comment:
One way is just to store an `byte[]` as `serializedConnectorOutputMetadataInfo`. The similar idea is used here: https://github.com/prestodb/presto/blob/aa78e33c3d22e372f2af9e94b78b05b097fc0cb1/presto-main/src/main/java/com/facebook/presto/metadata/Metadata.java#L242

where a `List<Slice> fragments` are some serialized connector specific information.